packages/next/compiled/use-subscription/index.js

```diff
@@ -0,0 +1,25 @@
+(()=>{"use strict";var e={800:e=>{
```

Comment:
This does not work on IE: `=>`

This code should be transpiled

Reply:
This was updates in https://github.com/vercel/next.js/pull/36159 which is available in the latest canary of Next.js